torch/csrc/distributed/autograd/context/dist_autograd_container.cpp

```diff
@@ -9,6 +9,7 @@ namespace autograd {
 constexpr int kAutoIncrementBits = 48;
 constexpr int64_t kAutoIncrementMask = (1LL << kAutoIncrementBits) - 1;
 constexpr int kMaxWorkerId = 65535;
+const std::chrono::duration<double> kContextTimeout = std::chrono::seconds(60);
```

Comment:
What's the current autograd timeout? Is 60s going to be too short of a timeout in some cases? I'm imaging the use case of doing long-running async RPCs within an autograd context. @pritamdamania87 any suggestions on this timeout?